building_energy_standards_data/applications/create_openstudio_standards_json.py

```diff
@@ -294,7 +304,7 @@ def create_openstudio_standards_space_data_json(
     # Export retrieved data
     if len(space_types) > 0:
         with open(
-            f"../../lib/openstudio-standards/standards/{code}/{code}_{code_version}/data/{code}_{code_version}.space_types.json",
+            f"{osstd_repository_path}/lib/openstudio-standards/standards/{code}/{code}_{code_version}/data/{code}_{code_version}.space_types.json",
```

Comment:
Will this blocks the generation of json files for other use cases? Maybe we should add a note to the comments of this function, make it specific that the function only works when there is a local openstudio standard repository exist and the path is provided?

Reply:
Sounds good, I made some changes to the comments.